web3rpc/rpc-specs/code-samples/java/src/main/java/opensdk/sdk/apis/admin/AdminAddPeerExample.java

```diff
@@ -1,14 +1,11 @@
-package opensdk.sdk.apis.admin;
-
-import opensdk.sdk.apis.constant.UrlConstants;
+let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
```

Comment:
@Sotatek-HauTran3 Please double check. It looks like ethers-ext code

Reply:
It's seem there are some mistake when using IDE for replace, I will fix it